src/core/client/framework/lib/bootstrap/createContext.tsx

```diff
@@ -118,6 +119,8 @@ export default async function createContext({
     postMessage: new PostMessageService(),
     localStorage: createLocalStorage(),
     sessionStorage: createSessionStorage(),
+    pymLocalStorage: pym && createPymStorage(pym, "localStorage"),
```

Comment:
Does this constructor `thing && thing` not possibly result in `false` versus undefined?

Reply:
It won't, also typescript would have complained :-)

Reply:
`thing1 && thing2` if anything is falsey, it returns that particular falsey value e.g. `null` or `undefined` or `false` or ""